core/src/main/java/kafka/log/remote/RemoteLogManager.java

```diff
@@ -1830,12 +1872,14 @@ void doHandleFollowerPartition(TopicIdPartition topicPartition) {
             expirationRLMTaskWithFuture.cancel();
         }
 
-        followerRLMTasks.computeIfAbsent(topicPartition, topicIdPartition -> {
-            RLMFollowerTask task = new RLMFollowerTask(topicIdPartition);
-            LOGGER.info("Created a new follower task: {} and getting scheduled", task);
-            ScheduledFuture<?> future = followerThreadPool.scheduleWithFixedDelay(task, 0, delayInMs, TimeUnit.MILLISECONDS);
-            return new RLMTaskWithFuture(task, future);
-        });
+        if (!remoteCopyDisabled) {
```

Comment:
I think we may have to leave the followerRLMTask to be running in the background. Assume that the follower gets reassigned from one broker to another when remoteCopy is disabled. Will the newly reassigned follower able to serve the remote data when consumer is configured to read from follower using the `preferredReadReplica` feature?

Reply:
hmm... good point. We need the follower task to get the correct remote offset.